### PR TITLE
provide clarification around using sum

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -1031,6 +1035,10 @@ positive, the sum should never get smaller; so if the sum does get
 smaller, the application will know that one of the situations listed
 above has happened.
 
+Despite the name sum, the sum field is not useful for applications
+that totally up the number of times that an event happened or
```
Comment (akeranen) [inline draft-ietf-core-senml.md]: s/totally up/maintain a running count of/

**Comment by akeranen** [comment]
PR #116 proposes fixes

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -371,7 +371,8 @@ Value:
 
 Sum:
 : Integrated sum of the values over time. Optional. This field is
-  in the unit specified in the Unit value multiplied by seconds.
+  in the unit specified in the Unit value multiplied by seconds. Due
+  to historical reason it is named sum instead of integral.
```
Comment (cabo) [inline draft-ietf-core-senml.md]: "For historical reasons"

File: draft-ietf-core-senml.md [hunk 3]
```diff
@@ -988,7 +989,10 @@ from 19th to the last:
 
 The measurements support sending both the current value of a sensor as
 well as the an integrated sum. For many types of measurements, the sum
-is more useful than the current value. For example, an electrical
+is more useful than the current value.
+For historical reasons, the sum field is called a sum instead of
```
Comment (cabo) [inline draft-ietf-core-senml.md]: »this field is called "sum"« maybe?

File: draft-ietf-core-senml.md [hunk 4]
```diff
@@ -988,7 +989,10 @@ from 19th to the last:
...
+integral which more accurately describes it's function.
```
Comment (cabo) [inline draft-ietf-core-senml.md]: its

**Review by cabo — CHANGES_REQUESTED** [review]

**Pull request merged**